packages/cicero-core/src/template.js

```diff
@@ -172,15 +174,58 @@ class Template {
         return hasher.digest('hex');
     }
 
+    /**
+     * signs a string made up of template hash and time stamp using private key derived
+     * from the keystore
+     * @param {string} [path] - path of the keystore to be used
+     * @param {string} [password] - password for the keystore file
+     * @return {object} - object containing signatory's metadata, timestamp, signatory's certificate, signature
+     * @private
```

Comment:
@jeromesimeon should I keep this method private? Also, once contract lifecycle is fixed I believe we might shift the signing and verifying logic to some other file like `utility.js`?

Reply:
@sanketshevkar private looks fine to me for now.